app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not command:
+        return
+    command_list = command.split()
+    if command_list[0] == "mv" and len(command_list) == 3:
+        if "/" in command_list[2]:
```

Comment:
`mv, file_name, new_file_name = command_list`
use unpacking to avoid code duplication